src/Components/Form/Form.js

```diff
@@ -91,16 +84,30 @@ export class Form extends Component {
     })
   }
 
+  handleSubmission = () => {
+    //invoke app's method for showing suggested activity
+  }
+
+  determineNextOrSubmit = () => {
+    let button;
+    if (this.state.allAnswers.length === 0) {
+      return <button onClick={this.updateAllAnswers} className='next-button form-button'>next</button>
+    } else if (this.state.allAnswers.length === this.state.questionsPerActivity.length) {
+      button = <button onClick={this.handleSubmission} className='submit-button form-button'>submit</button>
+    }
+    if (this.state.questionsPerActivity && button === undefined) {
+      button = <button onClick={this.updateAllAnswers} className='next-button form-button'>next</button>
+    }
+    return button;
+  }
```

Comment:
I have been trying to think about how we can fix our current issue where `card games` and `board games` display the same question twice if selected together.

I was thinking that we could possibly adjust the order in which our choices are displayed for our first questions `what activity excites you?` to have the card / board games questions be the last ones. We could add some logic to this function which evaluates if we will be rendering the duplicate question and instead render the submit button for that choice.

Still trying to think of decent logic but will let you know if I do

Reply:
I think that's a solid idea to try!  We could also consider lumping board games and card games as a single option too...